### ISSUE TITLE
Cannot Install Certified Chamberlain Plugin

### ISSUE THREAD

**Issue by gwoodbridge** [hoobs-org/HOOBS#374]
**Description**
When trying to install the HOOBS Certified Chamberlain plugin, it does not successfully load.

**Expected behavoir**
It should successfully load the plugin

**Plugin name**
@hoobs/chamberlain

**Version**
3.1.12

**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [ ] Yes
- [X] No
New Install

**Previous version**
N/A

**Did you originally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [X] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [ ] HOOBS SD Card
- [X] Custom Install

**If this is a custom install, what's your operating system?**
Raspbian Stretch on a Raspberry Pi 3 Model B

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
{
    "server": {
        "port": 8080,
        "origin": "*",
        "autostart": 0,
        "home_setup_id": "X-HM://0023ISYWYDOK6",
        "polling_seconds": 5
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-light",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "73044"
    },
    "bridge": {
        "name": "HOOBS",
        "port": 51826,
        "pin": "031-45-154",
        "username": "F0:39:CC:77:E0:49"
    },
    "description": "",
    "ports": {},
    "accessories": [],
    "platforms": []
}
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
+ @hoobs/chamberlain@1.0.8
added 5 packages from 5 contributors and audited 617 packages in 10.661s
found 0 vulnerabilities
Cannot find module 'chamberlain'
Unable to determine plugin type for "chamberlain"
[12/26/2019, 9:56:11 PM] Unhandled Rejection "Error: Cannot find module 'chamberlain'".
(node:2011) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 6)
[List Accessories] connect ECONNREFUSED 127.0.0.1:51826
```

**Additional context**
If you leave the plugin install page and go to the Installed Plugins tab, the plugin does show up and allow me to add a device and configure it. However, when attempting to start the service, I get the following errors:

```
[Homebridge] [12/26/2019, 9:38:22 PM] Loaded config.json with 1 accessories and 0 platforms.
[Homebridge] [12/26/2019, 9:38:22 PM] ---
[Homebridge] [12/26/2019, 9:38:22 PM] No plugins found. See the README for information on installing plugins.
[Homebridge] [12/26/2019, 9:38:22 PM] Loading 0 platforms...
[Homebridge] [12/26/2019, 9:38:22 PM] Loading 1 accessories...
[Homebridge] [12/26/2019, 9:38:22 PM] Error: The requested accessory 'Chamberlain' was not registered by any plugin.
[Homebridge] at API.accessory (/home/pi/.hoobs/node_modules/@hoobs/homebridge/lib/api.js:45:19)
[Homebridge] at Server._loadAccessories (/home/pi/.hoobs/node_modules/@hoobs/homebridge/lib/server.js:305:48)
[Homebridge] at Server.run (/home/pi/.hoobs/node_modules/@hoobs/homebridge/lib/server.js:81:14)
[Homebridge] at module.exports (/home/pi/.hoobs/node_modules/@hoobs/homebridge/lib/cli.js:86:12)
[Homebridge] at Object.<anonymous> (/home/pi/.hoobs/node_modules/@hoobs/homebridge/bin/homebridge:17:22)
[Homebridge] at Module._compile (internal/modules/cjs/loader.js:778:30)
[Homebridge] at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
[Homebridge] at Module.load (internal/modules/cjs/loader.js:653:32)
[Homebridge] at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
[Homebridge] at Function.Module._load (internal/modules/cjs/loader.js:585:3)
[Homebridge] [12/26/2019, 9:38:22 PM] Got SIGTERM, shutting down service...
```

**Comment by mkellsy** [comment]
Was not using the scoped plugin name. Fixed in 3.1.14.

**Comment by gwoodbridge** [comment]
Updated to 3.1.14 and everything works as expected. Thanks!
